### DISCUSSION TITLE
Icon not Clicable.

### DISCUSSION THREAD

**Discussion by caiocalhau** [lawnstarter/react-native-picker-select#525] [Q&A]
I've been using the Picker, but on android the icon it's not working when trying to press it, on iOS it's working fine. Does anyone have an solution for that?

**Comment by adrianso** [comment]
Have you tried setting `fixAndroidTouchableBug` to true? Seems to work for me.

**Reply by caiocalhau** [reply]
Thank's, that worked perfectly.
